about/ourmembers.md

```diff
@@ -1,18 +1,18 @@
 # Our Members
-We gratefully acknowledge the support of the following 232 member institutions.
+We gratefully acknowledge the support of the following 239 member institutions.
```

Comment:
I count 10 new institutions and societies as part of this commit, but it could be that the original count was off :-)